tests/testAutopas/tests/containers/AllContainersTests.cpp

```diff
@@ -0,0 +1,149 @@
+/**
```

Comment:
I think it would make more sense to test this using the AutoPas interface instead of the actual containers.
@FG-TUM what do you think?
the only problem with that is that these here, look more like unit tests and not tests of the entire thing...

this also means that, e.g., that tests of the actual containers are more error-prone (as these do no longer detect these exceptions)